src/cryptoadvance/specter/rpc.py

```diff
@@ -240,6 +253,8 @@ def wallet(self, name=""):
             path="{}/wallet/{}".format(self.path, name),
             timeout=self.timeout,
             session=self.session,
+            proxy_url=self.proxy_url,
```

Comment:
inject the procy_url into the wallet? Can't we avoid that? The wallet-implementation should imho be a bit more independent of the underlying network-technology.

Reply:
Bullshit. That's not the wallet-impl.